knowledge_graph/make_graph.py

```diff
@@ -188,48 +141,7 @@ def remove_edge_properties_from_nodes(G, to_remove):
             for node in list(G.nodes[node]["properties"][prop])
             if node not in list(to_delete)
         ]
-
-
-def save_graph_to_pickle(G, outfile_path):
-    file_path = os.path.join(outfile_path, "Climate_Mind_DiGraph.gpickle")
-    with open(file_path, "wb") as outfile:
-        nx.write_gpickle(G, outfile)
-
-
-def save_graph_to_gexf(G, outfile_path):
-    file_path = os.path.join(outfile_path, "Climate_Mind_DiGraph.gexf")
-    with open(file_path, "wb") as outfile:
-        nx.write_gexf(G, outfile)
-
-
-def save_graph_to_gml(G, outfile_path):
-    file_path = os.path.join(outfile_path, "Climate_Mind_DiGraph.gml")
-    with open(file_path, "wb") as outfile:
-        nx.write_gml(G, outfile)
-
-
-def save_graph_to_graphml(G, outfile_path):
-    file_path = os.path.join(outfile_path, "Climate_Mind_DiGraph.graphml")
-    with open(file_path, "wb") as outfile:
-        nx.write_graphml(G, outfile)
-
-
-def save_graph_to_yaml(G, outfile_path):
-    file_path = os.path.join(outfile_path, "Climate_Mind_DiGraph.yaml")
-    with open(file_path, "w") as outfile:
-        nx.write_yaml(G, outfile)
-
-
-def save_graph_to_json(G, outfile_path):
-    file_path = os.path.join(outfile_path, "Climate_Mind_DiGraph.json")
-    with open(file_path, "w") as outfile:
-        outfile.write(json_graph.jit_data(G, indent=4))
-
-
-def save_test_ontology_to_json(G, outfile_path):
-    file_path = os.path.join(outfile_path, "Climate_Mind_Digraph_Test_Ont.json")
-    with open(file_path, "w") as outfile:
-        outfile.write(json_graph.jit_data(G, indent=4))
+        # DM: uh... won't `node not in list(to_delete)` always evaluate to false?
```

Comment:
I'm pretty sure it doesn't always evaluate to false. Otherwise I wouldn't have wrote it into the code.

Reply:
Ok, I see where I was confused. I'll clarify this to use different variables names to clarify the "node" scope confusion. I'm thinking something like:

    G.nodes[node]["properties"][prop] = [
        value
        for value in list(G.nodes[node]["properties"][prop])
        if node not in list(to_delete)
    ]

Reply:
I think instead of 'value' should use 'partnerNode'.